### ISSUE TITLE
Regression: can't email long email addresses

### ISSUE THREAD

**Issue by bradvogel** [andris9/smtp-connection#32]
Customers using my product are complaining today that they can't send to long auto-reply email addresses such as:
2.76502047696709.12649470900136.76502049898901.4mtgutesvvbpnqn5wzg13g81kw8cn7xk4wz6nffte6jyjb7hx8@mail.asana.com
reply+000c89caea14b0bf53caf17fd0c9c67fce378f51fd1af05492cf000000011292f3d892a169ce07601530@reply.github.com

I traced this back to the introduction of the `isemail` library in:
https://github.com/andris9/smtp-connection/commit/21590db277a9b14f4efa25e968fbd91dbaadee0e#diff-3f54aaa87cd2e1c43e57bb854e2a0be7R570

I also filed https://github.com/hapijs/isemail/issues/13. But as a workaround, can you provide a bypass or consider removing `isemail`? It appears to be too strict. I'll be glad to file a PR.

**Comment by bradvogel** [comment]
Looks like this is a *technically* invalid, but still very common in the real world. Please consider removing this check or providing a bypass.

**Comment by wearhere** [comment]
I don't understand the reason to try to validate the email addresses beforehand. Isn't it client error if the email addresses are invalid? Clearly it is technically possible for nodemailer to use these addresses, since it did so prior to the introduction of isemail, and for GitHub and Asana to receive messages sent to them.

**Comment by andris9** [comment]
I agree that isEmail turned out to be too strict on validation. It was added due to potential security issue where specially crafted addresses might mess up the smtp transaction. Full compliancy is not required though in the context of smtp-connection, so I removed isEmail in v1.3.8 and from now on only check that the address does not include newline symbols and &lt; &gt; which are part of the SMTP syntax.
